docs/hassio_hass.md

```diff
@@ -41,17 +41,17 @@ Update the Hass.io component configuration in your `configuration.yaml` to point
 
 ```yaml
 # configuration.yaml
-hassio:
+frontend:
   development_repo: /home/paulus/dev/hass/home-assistant-polymer
 ```
 
 To build a local version of the Hass.io panel, go to the frontend repository and run:
 
 ```bash
-cd hassio
-script/build_hassio
+script/bootstrap
+script/develop
```

Comment:
This is wrong too. You are doing the frontend dev instructions, not the one for the Hass.io panel.

Reply:
It should be `script/develop` from inside the hassio dir.

Reply:
Switched this back to `cd hassio` then script/develop. 👍